framework/ChiPhysics/PhysicsMaterial/transportxsections/prop_transportxs_01_readfromchi.cc

```diff
@@ -486,82 +497,111 @@ void chi_physics::TransportCrossSections::
       auto& f  = file;
       auto& fw = word;
 
+      //============================================================
+      // Group Structure Data
+      //============================================================
+
       if (fw == "GROUP_STRUCTURE_BEGIN")
         ReadGroupStructure("GROUP_STRUCTURE",
                            e_bounds, num_groups, f, ls, ln);
 
+      if (fw == "INV_VELOCITY_BEGIN")
+        Read1DData("INV_VELOCITY", inv_velocity, num_groups, f, ls, ln);
+      if (fw == "VELOCITY_BEGIN" &&
+          std::all_of(inv_velocity.begin(), inv_velocity.end(),
+                      [](double x) { return x == 0.0; }))
```

Comment:
Exactly! This reads "Are all of the elements of `inv_velocity` zero?". I'm glad you asked for this particular case because this needed to be changed to a check for `inv_velocity` being empty since we no longer allocate by default!